lib/user.js

```diff
@@ -100,6 +100,9 @@ function role_permissions(roles, callback) {
  * implementation always require the account parameter to be set.
  */
 function access(permission, account, callback) {
+  var i,
```

Comment:
Not sure this variable declarations are at the right place.
+ `var i` belongs inside `findPermission()`
+ `var permissionsFound` is redundant if `findPermission()` return a boolean.